app/src/main/java/io/gnosis/safe/ui/settings/app/passcode/RepeatPasscodeFragment.kt

```diff
@@ -84,7 +85,14 @@ class RepeatPasscodeFragment : BaseViewBindingFragment<FragmentPasscodeBinding>(
                             },
                             confirmCallback = { dialog ->
                                 viewModel.enableBiometry()
```

Comment:
I would place this call after viewModel.encryptPasscodeWithBiometricKey. This sequence seems more correct to me. Biometric is enabled only after all necessary work is done.

Reply:
This way `settingsHandler.useBiometrics = false` should not be necessary